src/utils/daos_hdlr.c

```diff
@@ -3096,23 +3065,20 @@ dm_disconnect(struct cmd_args_s *ap,
 		if (is_posix_copy) {
 			rc2 = dfs_umount(dst_file_dfs->dfs);
 			if (rc2 != 0) {
-				fprintf(ap->errstream,
-					"failed to unmount destination (%d)\n", rc2);
+				DH_PERROR_DER(ap, rc2, "failed to unmount destination");
```

Comment:
SYS, not DER for dfs_umount
```suggestion
				DH_PERROR_SYS(ap, rc2, "failed to unmount destination");
```

Reply:
Thank you.  The test should have picked this up...